acceptance/suites/pre_suite/foss/30_install_dev_repos.rb

```diff
@@ -26,11 +26,12 @@
   end
 end
 
-if puppetdb_supported_platforms.include?(master.platform) then
-  step "Install PuppetDB repository" do
-    install_puppetlabs_dev_repo(
-      master, 'puppetdb', test_config[:puppetdb_build_version],
-      repo_config_dir, install_opts)
+puppetdb_supported_platforms.each do |pdb_platform|
+  if master.platform.include?pdb_platform.source then
```

Comment:
If I do the following in irb...

~~~ruby
if "el-6-x86_64" =~ /el/ then
  puts "matches"
end
~~~

... "matches" is written out.  If I do ...

~~~ruby
if "debian-7" =~ /el/ then
  puts "matches"
end
~~~

... "matches" is not written out.

Seems like `master.platform =~ pdb_platform` returning zero would be okay here?